src/extensions/menu.extension.ts

```diff
@@ -17,8 +17,20 @@ export class Menu {
     options?: {
       level?: number;
       headline?: string;
+      showHelp?: boolean;
+      helpMessage?: string;
+      backMessage?: string;
+      cancelMessage?: string;
+      byeMessage?: string;
     }
   ) {
+    const messages = {
+      help: options.helpMessage || '[ help ]',
```

Comment:
The parameter `options` is optional.  If it is not set, an error will occur here, because in `options.helpMessage`, for example, an attempt is made to access the `helpMessage` property in the `options` object.  But `options` is `null` in the example, so this will result in `null.helpMessage` and cause an error. 

We can prevent this by inserting the following line, for example, beforehand: 
```
options = options || {};
```

Reply:
Fixed!